libraries/botbuilder-dialogs-adaptive-runtime-integration-express/src/index.ts

```diff
@@ -9,26 +9,34 @@ import type { ActivityHandlerBase, BotFrameworkAdapter, ChannelServiceRoutes } f
 import { Configuration, getRuntimeServices } from 'botbuilder-dialogs-adaptive-runtime';
 import type { ServiceCollection } from 'botbuilder-dialogs-adaptive-runtime-core';
 
+// Explicitly fails checks for `""`
+const NonEmptyString = t.String.withConstraint((str) => str.length > 0 || 'must be non-empty string');
```

Comment:
Is `NonEmptyString` something we should keep somewhere more "top-level" in the SDK and export it? It seems like something we'd use frequently.

Reply:
Eventually I think we should have some conventions around runtypes expressed. At the moment, I'm not sure what pattern I want to use so I'm hesitant to start extracting common things just yet.